client/src/components/SignUpPage2.js

```diff
@@ -0,0 +1,198 @@
+import React from "react";
+import { Paper, Grid } from "@material-ui/core";
+import Button from "@material-ui/core/Button";
+import { useTheme, makeStyles } from "@material-ui/core/styles";
+import RoomOutlinedIcon from "@material-ui/icons/RoomOutlined";
+import SignInModalFooter from "./MuiComponents/SignInModalFooter";
+import SignInModalHeader from "./MuiComponents/SignInModalHeader";
+import CloseModal from "./MuiComponents/CloseModal";
+
+const signUp2Styles = makeStyles((theme) => ({
+	sidePadding: {
+		padding: "0rem 1rem 0rem 1rem",
+	},
+	paper: {
+		position: "relative",
+		overflow: "hidden",
+		width: "500px",
+	},
+
+	locationPaper: {
+		margin: "0.5rem 0 0.5rem 0",
+	},
+	locationText: {
+		marginLeft: "1.25rem",
+		fontWeight: "bold",
+	},
+	locationIcon: {
+		paddingLeft: "0.75rem",
+		color: `${theme.palette.secondary.main}`,
+	},
+	pr1: {
+		paddingRight: "1rem",
+	},
+
+	mt2: {
+		marginTop: "2rem",
+	},
+	signUpButton: {
+		color: "white",
+		fontSize: "16px",
+		borderRadius: "6px",
+		marginTop: "2.5rem",
+		padding: "0.5rem, 1rem, 0.5rem, 1rem",
+	},
+	secondary: {
+		color: `${theme.palette.secondary.main}`,
+	},
+}));
+
+function SignUpPage2() {
```

Comment:
what's signuppage2?

Reply:
Its the second part of Sign Up where user selects the location (which is static for now). I made them into two separate components.